src/xmidtsend_rbus.c

```diff
@@ -267,6 +267,21 @@ int validateXmidtData(wrp_msg_t * eventMsg, char **errorMsg, int *statusCode)
 		ParodusError("errorMsg: %s, statusCode: %d\n", *errorMsg, *statusCode);
 		return 0;
 	}
+	else
+	{
+		ParodusPrint("Validate content_type\n");
+		if ( (strcmp(eventMsg->u.event.content_type, "application/json") == 0) || (strcmp(eventMsg->u.event.content_type, "avro/binary") == 0) || (strcmp(eventMsg->u.event.content_type, "application/msgpack") == 0) || (strcmp(eventMsg->u.event.content_type, "application/binary") == 0) )
```

Comment:
Can we define an enum or string array, so we could add to it and check all in the list

Reply:
Yes, that can be done, will try with string array